jxl/src/headers/frame_header.rs

```diff
@@ -347,11 +348,21 @@ pub struct FrameHeader {
     #[coder(u2S(Bits(8), Bits(11) + 256, Bits(14) + 2304, Bits(30) + 18688))]
     #[default(0)]
     #[condition(have_crop && frame_type != FrameType::ReferenceOnly)]
-    pub x0: i32,
+    x0_raw: u32,
 
     #[coder(u2S(Bits(8), Bits(11) + 256, Bits(14) + 2304, Bits(30) + 18688))]
     #[default(0)]
     #[condition(have_crop && frame_type != FrameType::ReferenceOnly)]
+    y0_raw: u32,
+
+    #[coder(Bits(0))]
+    #[default(unpack_signed(x0_raw))]
+    #[condition(false)]
+    pub x0: i32,
+
+    #[coder(Bits(0))]
+    #[default(unpack_signed(y0_raw))]
+    #[condition(false)]
```

Comment:
funny that this pattern was never used before elsewhere in the header parsing

Reply:
actually, is this even necessary? Isn't https://github.com/libjxl/jxl-rs/blob/4c123a36847b2fa0f6ec731028b3a86e219bfcb4/jxl/src/headers/encodings.rs#L95 doing the correct thing already